README.md

```diff
@@ -8,14 +8,6 @@ VB365 Calculator Scripts - Gather VB365 data for use in the https://calculator.v
 
 * [MIT License](LICENSE)
 
-## ✍ Contributions
-
-We welcome contributions from the community! We encourage you to create [issues](https://github.com/VeeamHub/veeam-calculators/issues/new/choose) for Bugs & Feature Requests and submit Pull Requests. For more detailed information, refer to our [Contributing Guide](CONTRIBUTING.md).
-
-## 🤝🏾 License
-
-* [MIT License](LICENSE)
-
 ## 🤔 Questions
 
-If you have any questions or something is unclear, please don't hesitate to [create an issue](https://github.com/VeeamHub/veeam-calculators/issues/new/choose) and let us know!
\ No newline at end of file
+If you have any questions or something is unclear, please don't hesitate to [create an issue](https://github.com/VeeamHub/{repo-name}/issues/new/choose) and let us know!

```

Comment:
@shapedthought This change will break functionality for this link. The previous value was correct:

`https://github.com/VeeamHub/veeam-calculators/issues/new/choose`